cmd/mimir/config-descriptor.json

```diff
@@ -7819,6 +7819,28 @@
               "fieldFlag": "blocks-storage.tsdb.block-postings-for-matchers-cache-force",
               "fieldType": "boolean",
               "fieldCategory": "experimental"
+            },
+            {
+              "kind": "field",
+              "name": "forced_head_compaction_min_in_memory_series",
+              "required": false,
+              "desc": "When the number of in-memory series in the ingester is equal or greater than this setting, the ingester tries to force the TSDB Head compaction. The forced compaction removes from the memory all samples and inactive series up until -ingester.active-series-metrics-idle-timeout time ago. After a forced compaction, the ingester will not accept any sample with timestamp older than -ingester.active-series-metrics-idle-timeout time ago (unless out of order ingestion is enabled). The ingester checks every -blocks-storage.tsdb.head-compaction-interval whether a force compaction is required. 0 to disable.",
```

Comment:
```suggestion
              "desc": "When the number of in-memory series in the ingester is equal to or greater than this setting, the ingester tries to force the TSDB Head compaction. The forced compaction removes from the memory all samples and inactive series up until -ingester.active-series-metrics-idle-timeout time ago. After a forced compaction, the ingester will not accept any sample with a timestamp older than -ingester.active-series-metrics-idle-timeout time ago (unless out of order ingestion is enabled). The ingester checks every -blocks-storage.tsdb.head-compaction-interval whether a force compaction is required. Use `0` to disable it.",
```

Reply:
The use of ticks here will not format it correctly when generating the config doc. I will pick the rest of the suggestion tho.